projects/ngx-d3timeline/src/lib/store/state.ts

```diff
@@ -0,0 +1,26 @@
+import { TimelineEvent } from '../timeline-event';
+import { EventRectangleDragEvent } from '../content/event-rectangle-drag-event';
+import { TimelineView } from '../view/timeline-view';
+import { TimeScale, BandScale } from '../scale-types';
+import { AxisOrientations } from '../axis-orientations';
+
+export interface State {
+  data: TimelineEvent[];
```

Comment:
more logical ordering might be:

view
axisOrientations
data
timeScale
bandScale
dragEvent

OR

data
view
axisOrientations
timeScale
bandScale
dragEvent